src/AccountServiceProvider.php

```diff
@@ -22,6 +23,8 @@ public function boot()
         $this->publishes([
             __DIR__.'/../resources/views' => resource_path('views/vendor/rapidez'),
         ], 'views');
+
+        Blade::componentNamespace('Rapidez\\Account\\View\\Components', 'rapidez');
```

Comment:
Is this really needed? https://laravel.com/docs/master/packages#view-components implies it's only needed when it's non-conventional, most likely because it's `View` instead of `Views`?